frontend/pendingTranslations.ftl

```diff
@@ -23,7 +23,7 @@ fx-desktop-2 = { -brand-name-firefox } for Desktop
 fx-mobile-2 = { -brand-name-firefox } for Mobile
 fx-containers = { -brand-name-firefox } Containers
 
-## Phone Masking FAQ 
+## Phone Masking FAQ
```

Comment:
nit: whitespace edit

```suggestion
## Phone Masking FAQ 
```

Reply:
Ah, I set up my editor to autoremove trailing whitespace from Fluent files because that'll get rejected when submitting it to the l10n repository. I've asked everyone to do the same in their editor so that won't make it in anymore in the first place - do you mind if I leave this in here? Or shall I submit a separate PR to remove trailing whitespace?